src/CommandingOfficers/AWBW/GE/JessDSBW.java

```diff
@@ -0,0 +1,115 @@
+package CommandingOfficers.AWBW.GE;
+
+import java.util.ArrayList;
+import CommandingOfficers.*;
+import CommandingOfficers.CommanderAbility.CostBasis;
+import CommandingOfficers.AWBW.AWBWCommander;
+import Engine.GameScenario;
+import Engine.Combat.StrikeParams;
+import Engine.UnitMods.UnitDamageModifier;
+import Engine.UnitMods.UnitModifier;
+import Engine.UnitMods.UnitMovementModifier;
+import Engine.UnitMods.UnitTypeFilter;
+import UI.UIUtils;
+import Terrain.MapMaster;
+import Units.Unit;
+import Units.UnitModel;
+
+public class JessDSBW extends AWBWCommander
+{
+  private static final long serialVersionUID = 1L;
+
+  private static final CommanderInfo coInfo = new instantiator();
+  public static CommanderInfo getInfo()
+  {
+    return coInfo;
+  }
+  private static class instantiator extends CommanderInfo
+  {
+    private static final long serialVersionUID = 1L;
+    public instantiator()
+    {
+      super("Jess", UIUtils.SourceGames.AWBW, UIUtils.GE, "3BW");
+      infoPages.add(new InfoPage(
+            "AWDS Jess for AWBW\n"
+          + "Jess, but less terrible.\n"
```

Comment:
This was amusing to write, I'm sure, but doesn't add anything in the CO description.